File: Week 1/Data Structures and Algorithms/ECommerceSearch/Code/Product.java
public class Product {
    int productId;
    String productName;
    String category;
    public Product(int id, String name, String category) {
        this.productId = id;
        this.productName = name;
        this.category = category;
    }
    @Override
    public String toString() {
        return productId + " - " + productName + " (" + category + ")";
    }
}

File: Week 3 - 6365294/Entity Framework Core/Lab 1/Product.cs
using lab1;

namespace lab1
{
    public class Product
    {
        public int ProductId { get; set; }
        public string Name { get; set; }
        public int Quantity { get; set; }

        public int CategoryId { get; set; }
        public Category Category { get; set; }
    }
}
